### DISCUSSION TITLE
adding Intl.ListFormat to useFormatter

### DISCUSSION THREAD

**Discussion by stefanprobst** [amannn/next-intl#306] [Ideas]
would you be open to adding [`Intl.ListFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat) to `useFormatter()`?

**Comment by amannn** [comment]
Generally, yes—absolutely! I didn't have time to look deeper into it yet though.

Are you interested in contributing this feature? Would you like to write up a feature issue that describes the API and possible considerations?

**Reply by stefanprobst** [reply]
> write up a feature issue that describes the API and possible considerations

yes, will do!
